src/api/v3/feedback/search_result_feedback.py

```diff
@@ -0,0 +1,13 @@
+from django.http import HttpResponse
+
+from api.v3.feedback import feedback_handler
+from cms.models import SearchResultFeedback
+
```

Comment:
Disable csrf-protection for the feedback api:
```suggestion


@csrf_exempt
```